.github/workflows/create-release.yml

```diff
@@ -0,0 +1,101 @@
+      run: |
+        version="${{ github.event.client_payload.version }}"
+        if [ -z "${version}" ]; then
+          version="${{ github.event.inputs.version }}"
+        fi
+        echo "version=${version}" >> "$GITHUB_OUTPUT"
+
+    - name: Checkout cflinuxfs4
+      uses: actions/checkout@v3
+      with:
+        repository: cloudfoundry/cflinuxfs4
+        # TODO - set this to main
+        ref: c4126e9cc86d660c6444c3eb50149e3cfdc1eb8f
+        path: cflinuxfs4
+
+    - name: Create stack
+      id: create-stack
+      env:
+        DOCKERHUB_USERNAME: ${{ secrets.CF_DOCKERHUB_USERNAME }}
+        DOCKERHUB_PASSWORD: ${{ secrets.CF_DOCKERHUB_PASSWORD }}
+      run: |
+        pushd $GITHUB_WORKSPACE/cflinuxfs4
+          echo "${DOCKERHUB_PASSWORD}" | sudo skopeo login --username "${DOCKERHUB_USERNAME}" --password-stdin index.docker.io
+          echo "Making compat stack"
+          make compat version=${{ steps.repo-dispatch.outputs.version }}
+        popd
+
+    - name: BOSH release creation
+      id: bosh-release
+      # TODO change this to main
+      uses: cloudfoundry/buildpacks-github-config/actions/stack/create-release@bosh-release
+      with:
+        version: ${{ steps.repo-dispatch.outputs.version }}
+        access-key-id: ${{ secrets.CF_S3_ACCESS_KEY }}
+        secret-access-key: ${{ secrets.CF_S3_SECRET_ACCESS_KEY }}
+        release-name: cflinuxfs4-compat
+        blob-glob: cflinuxfs4/cflinuxfs4-*.tar.gz
+        blob-name: rootfs
+
+    - name: Git commit blobs.yml
+      id: commit-1
+      uses: paketo-buildpacks/github-config/actions/pull-request/create-commit@main
+      with:
+        message: "Updating blobs for cflinuxfs4-compat bosh release version ${{ needs.create.outputs.version }}"
+        pathspec: "config/blobs.yml"
+        # TODO
+        keyid: ${{ secrets.CF_BOT_GPG_SIGNING_KEY_ID }}
+        key: ${{ secrets.CF_BOT_GPG_SIGNING_KEY }}
+
+    - name: Git commit
+      id: commit-2
+      uses: paketo-buildpacks/github-config/actions/pull-request/create-commit@main
+      with:
+        message: "Final cflinuxfs4-compat bosh release version ${{ needs.create.outputs.version }}, containing cflinuxfs-compat version ${{ needs.create.outputs.version }}"
+        pathspec: ".final_builds releases/**/*-${{ needs.create.outputs.version }}.yml releases/**/index.yml"
+        # TODO
+        keyid: ${{ secrets.CF_BOT_GPG_SIGNING_KEY_ID }}
+        key: ${{ secrets.CF_BOT_GPG_SIGNING_KEY }}
+
+    - name: Git push
+      if: ${{ steps.commit-1.outputs.commit_sha != '' || steps.commit-2.outputs.commit_sha != '' }}
+      uses: paketo-buildpacks/github-config/actions/pull-request/push-branch@main
+      with:
+        branch: "main"
+
+    - name: Create Release
+      uses: paketo-buildpacks/github-config/actions/release/create@main
+      with:
+        repo: ${{ github.repository }}
+        token: ${{ secrets.CF_BOT_GH_TOKEN }}
+        tag_name: v${{ steps.repo-dispatch.outputs.version }}
```

Comment:
Why is there a v prefix? I don't believe any bosh releases have v prefix.
Also make sure "bosh release" in not made with v prefixed version

Reply:
I copied how we have it in cflinuxfs4: https://github.com/cloudfoundry/cflinuxfs4-release/releases/tag/v1.17.0

The tag has a `v` prefix, but the release name does not

Reply:
what do you mean by the bosh release comment?

Reply:
Oops typo. Edited

Reply:
I have removed the `v` in the latest commit